ana_scans.py

```diff
@@ -83,19 +86,18 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
       cmd.append("--fileScurveFitTree=%s"%(filename_Trim))
       pass
 
-    postCmds[0] = ["mkdir","-p","%s"%(elogPath)]
     postCmds[1] = ["cp","%s/ThresholdScanData/ThreshSummary.png"%(dirPath),
                    "%s/ThreshSummary_%s.png"%(elogPath,cName)]
     postCmds[2] = ["cp","%s/ThresholdScanData/ThreshPrunedSummary.png"%(dirPath),
                    "%s/ThreshPrunedSummary_%s.png"%(elogPath,cName)]
-    postCmds[3] = ["cp","%s/ThresholdScanData/chConfig_MasksUpdated.txt"%(dirPath),
-                   "%s/chConfig_MasksUpdated_%s.txt"%(elogPath,cName)]
-    postCmds[4] = ["cp","%s/ThresholdScanData/vfatConfig.txt"%(dirPath),
+    postCmds[3] = ["cp","%s/ThresholdScanData/vfatConfig.txt"%(dirPath),
                    "%s/vfatConfig_%s.txt"%(elogPath,cName)]
```

Comment:
This is okay for now, but we should discuss if this makes sense. These text files are not going to be posted to the elog ever, so maybe we should have a different place to put them.